SonarQube.Client.Tests/Models/SonarQubeIssueTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarQube.Client.Models;
+
+namespace SonarQube.Client.Tests.Models
+{
+    [TestClass]
+    public class SonarQubeIssueTests
+    {
+        [TestMethod]
+        public void Ctor_FilePathCanBeNull()
```

Comment:
Tests to show/check null handling. The first two tests are for pre-existing behaviours, the third checks that flows are never null.

Reply:
I think it's worth having not-null tests as well, e.g. that the passed flows are used.

Reply:
Ok but I think it's very low value.